relational_operators/tests/SortMergeRunOperator_unittest.cpp

```diff
@@ -1404,13 +1407,14 @@ class SortMergeRunOperatorTest : public ::testing::Test {
     std::size_t num_receieved = 0;
     do {
       if (bus_.ReceiveIfAvailable(foreman_client_id_, &msg)) {
-        EXPECT_EQ(kWorkOrderFeedbackMessage, msg.tagged_message.message_type());
-        WorkOrder::FeedbackMessage feedback_msg(
-            const_cast<void *>(msg.tagged_message.message()),
-            msg.tagged_message.message_bytes());
-        EXPECT_EQ(kOpIndex, feedback_msg.header().rel_op_index);
-        merge_op_->receiveFeedbackMessage(feedback_msg);
-        num_receieved++;
+        if (msg.tagged_message.message_type() == kWorkOrderFeedbackMessage) {
```

Comment:
My bad, it was a mix-up due to the merging master. I will fix it in a separate PR. Thanks for pointing it out.